postgres-protocol/src/message/backend.rs

```diff
@@ -901,6 +901,70 @@ impl<'a> Field<'a> {
     }
 }
 
+/// A struct representing the fields of a RowDescription message. Clones the fields to avoid keeping
+/// the `RowDescriptionMessage`.
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct ClonedField {
```

Comment:
I feel like `OwnedField` might be a better name

Reply:
Done